data/labels.json

```diff
@@ -0,0 +1,12 @@
+[
```

Comment:
Kindly add the file to .gitignore

Reply:
I wanted the app to start with default books, music albums... so the reviewer can see it straight from the console